README.md

```diff
@@ -7,10 +7,14 @@ WebDriver.
 
 ## Configure your Bazel project
 
-For all languages, you need to add the following to your MODULE.bazel file:
+For each language , you need to add the following to your MODULE.bazel file:
```

Comment:
For each language that is used by your project, you need to add the corresponding  sub module to your MODULE.bazel file: